### ISSUE TITLE
Error on opening MLT file with UTF-8, file not saved in UTF-8, LC_NUMERIC contains special char

### ISSUE THREAD

**Issue by Pathduck** [mltframework/shotcut#173]
When saving a project as an MLT file, since my locale is set to Norwegian (my Windows is in English though), the file is saved with the following:

```xml
<?xml version="1.0" encoding="utf-8"?>
<mlt LC_NUMERIC="Norwegian Bokmål_Norway.1252" version="0.9.9" title="Shotcut version 15.12.03" producer="playlist0">
```

However the file is not encoded in UTF-8 (according to notepad++), it's in Ansi. So XML parsing fails with:

[Error  ] <MLT> [producer_xml] parse fatal: Input is not proper UTF-8, indicate encoding !Bytes: 0xE5 0x6C 0x5F 0x4E	row: 2	col: 32

Row 2 col 32 is the Norwegian character 'å' .

Basically this leads to being unable to save projects being worked on.

Workaround:
- Remove LC_NUMERIC or set it to blank value: LC_NUMERIC=""
- Change file encoding to UTF-8 using for instance Notepad++.

**Comment by ddennedy** [comment]
I reproduce this. It is trying to to save UTF-8 as can be seen here: https://github.com/mltframework/mlt/blob/master/src/modules/xml/consumer_xml.c#L839

However, it only works if all the strings put into MLT or MLT retrieves is UTF-8. It appears the locale string we are getting from MinGW glibc Windows is not in UTF-8.

**Comment by Pathduck** [comment]
Thanks for looking into this Dan, hopefully this can be fixed in the next version :)

I'm just starting out with using Shotcut so I will be able to use the workaround until then.
Thanks!

**Comment by ddennedy** [comment]
Fixed in MLT git commit https://github.com/mltframework/mlt/commit/25491c7c592a9a0389f2329fcbae671aea24af01
